src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,69 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid("3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8");

```

Comment:
For both these guids, we probably want to avoid GUID string parsing.  I would replace it to the least with the one that take the hex values like [here](https://github.com/microsoft/CsWinRT/blob/9486d45237bbbfa2a3aadb2341dd0118164540dd/src/cswinrt/strings/ComInteropHelpers.cs#L91).

Reply:
> For both these guids, we probably want to avoid GUID string parsing. I would replace it to the least with the one that take the hex values like [here](https://github.com/microsoft/CsWinRT/blob/9486d45237bbbfa2a3aadb2341dd0118164540dd/src/cswinrt/strings/ComInteropHelpers.cs#L91).

Thank you for your suggestions in the code. I have adjusted the code according to your comments